test/e2e/upgrades/ingress.go

```diff
@@ -95,10 +95,9 @@ func (t *IngressUpgradeTest) Setup(f *framework.Framework) {
 	// Allocate a static-ip for the Ingress, this IP is cleaned up via CleanupGCEIngressController
 	t.ipName = fmt.Sprintf("%s-static-ip", ns.Name)
 	t.ip = t.gceController.CreateStaticIP(t.ipName)
-
```

Comment:
This seems unnecessary change here for readability.

Reply:
👍